network/throttling/common.go

```diff
@@ -20,10 +20,11 @@ type MsgByteThrottlerConfig struct {
 
 // Used by the sybil-safe inbound and outbound message throttlers
 type commonMsgThrottler struct {
-	log  logging.Logger
-	lock sync.Mutex
+	log      logging.Logger
+	subnetID ids.ID
```

Comment:
nit: shouldn't we be using the overridden manager here, rather than the combo subnetID + full validator manager?